core/templates/dev/head/pages/base.html

```diff
@@ -62,6 +62,8 @@
     <meta property="og:image" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/288x288_logo_mint.png')}}">
 
     <link rel="apple-touch-icon" href="/assets/images/logo/favicon.png">
+    <link rel="preload" href="/third_party/generated/webfonts/fa-solid-900.woff2" as="font" type="font/woff2">
+    <link rel="preload" href="/third_party/generated/webfonts/fa-brands-400.woff2" as="font" type="font/woff2">
```

Comment:
Please move these after line 37, also don't we use regular sized icons too and thus should preload them?

Reply:
Hi @apb7 and @vojtechjelinek,
`fa-solid-900.woff2` and `fa-brands-400.woff2 `are widely requested in the codebase that's why I've preloaded only them in base.html.

Also, e2e tests are falling with this error seems like while pre-loading it is unable to locate the hashed font files
`Expected $[0] = Entry({ level: SEVERE, message: 'http://localhost:9001/build/third_party/generated/webfonts/fa-solid-900.a7140145ebaaf5fb14e40430af5d25c4.woff2 - Failed to load resource: the server responded with a status of 404 (Not Found)', timestamp: 1559024798692, type: '' }) to equal undefined.`

Reply:
If you can't resolve this you can remove the preloads and I will add them later myself. I don't want to block you with stuff that's out of the scope of this PR.

Reply:
yes, i've reverted the changes